ccm_web/client/src/App.tsx

```diff
@@ -62,6 +64,12 @@ function App (): JSX.Element {
     )
   }
 
+  if (!globals.user.hasCanvasToken) {
+    // Initiate OAuth flow
+    location.href = '/canvas/redirectOAuth'
+    return loading
+  }
```

Comment:
I think it might be reasonable from a UX perspective to add a banner or alert here instead that links to `/canvas/redirect`, telling the user that the Canvas API integration is missing and that they need to go through the OAuth flow. With the changes I've made, this would also be show to the user the first time they access CCM. What do people think?

Reply:
It sounds like @janelilr and the UX team recommend some kind of explanatory interstitial page. I think this can be a separate issue for now; I have a prototype that I will share with the team that I can open a PR for after this is approved/merged.

Reply:
See #189.